src/main/java/com/team1678/frc2024/Constants.java

```diff
@@ -515,6 +519,44 @@ public static final class ClimberConstants {
 			kClimberServoConstants.kNeutralMode = NeutralModeValue.Brake;
 		}
 	}
+	/**
+	 * For standard deviations:
+	 * Lower = trust more, higher = trust less
+	 */
+	public static final class PhotonVisionConstants {
+		// Normal (but is inverted later because 3d rotations are silly)
+		public static double kCameraPitch = -15;
+		// Cameras are upside down
+		public static double kCameraRoll = 180;
```

Comment:
Is the camera upside down in the photonvision web app? I assumed you wouldn't have to do this if you corrected it via the web app.

Reply:
The pv web app does not support rotation for calibrated cameras

Reply:
Got verbal approval from nick